cli/src/cmd_docs.rs

```diff
@@ -52,8 +55,17 @@ fn to_json(cmd: &Command) -> JsonDoc {
         })
         .collect::<Vec<_>>();
     args.sort_unstable();
+
+    let name = cmd.get_name().to_string();
+    let version: Option<String> = if name == "oxide" {
```

Comment:
Why are we doing this conditionally? I've seen some people rename the command to different things

Reply:
Maybe there's a better way. I only want the version on the top-most command, without this it'd be on every command in the CLI right?

Reply:
I see; thanks, I didn't take the time to understand how this worked.